cmd/migration/v0.9.1-v1/mapper/vault_wallet.go

```diff
@@ -0,0 +1,118 @@
+package mapper
+
+import (
+	"fmt"
+
+	v091domain "github.com/tdex-network/tdex-daemon/cmd/migration/v0.9.1-v1/v091-domain"
+	v1domain "github.com/tdex-network/tdex-daemon/cmd/migration/v0.9.1-v1/v1-domain"
+)
+
+const (
+	FeeAccount = iota
+	WalletAccount
+	FeeFragmenterAccount
+	MarketFragmenterAccount
+	UnusedAccount3
+
+	nameSpaceFormat = "bip84-account%d'"
+)
+
+func (m *mapperService) FromV091VaultToV1Wallet(
+	vault v091domain.Vault,
+	walletPass string,
+) (*v1domain.Wallet, error) {
+	accounts := make(map[string]*v1domain.Account)
+	accountsByLabel := make(map[string]string)
+	highestAccountIndex := 0
+	masterKey, err := vault.MasterKey(walletPass)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, v := range vault.Accounts {
+		label, err := m.getLabel(v.AccountIndex)
+		if err != nil {
+			return nil, err
+		}
+
+		namespace := fmt.Sprintf(nameSpaceFormat, v.AccountIndex)
+		accountsByLabel[label] = namespace
+
+		if v.AccountIndex > highestAccountIndex {
+			highestAccountIndex = v.AccountIndex
+		}
+
+		xpub, err := v091domain.Xpub(
+			uint32(v.AccountIndex),
+			masterKey,
+		)
+		if err != nil {
+			return nil, err
+		}
+
+		accounts[namespace] = &v1domain.Account{
+			AccountInfo: v1domain.AccountInfo{
+				Namespace: namespace,
+				Label:     label,
+				Xpub:      xpub,
+				DerivationPath: fmt.Sprintf(
+					"%s/%d", v091domain.RootPath, v.AccountIndex,
```

Comment:
Watch out! Account must be hardened:
```suggestion
					"%s/%d'", v091domain.RootPath, v.AccountIndex,
```